ntp-client/src/Network/NTP/Client.hs

```diff
@@ -265,21 +304,6 @@ spawnNtpClient tracer settings ncStatus = do
             resolve
           else return addrs
 
--- |
--- Run NTP client in a separate thread; it returns a mutable cell which holds
--- @'NtpStatus'@.
---
--- This function should be called once, it will run an NTP client in a new
--- thread until the program terminates.
-withNtpClient :: Tracer IO NtpTrace -> NtpClientSettings -> IO (TVar NtpStatus)
```

Comment:
Provide a simlar top level function.  Maybe `withNtpClient :: Tracer IO NptTrace -> NtpClientSetting -> IO NtpClient`.
Including `Tracer` in `NtpClientSettings` might be a good idea too.

Reply:
It is already there. It has just been renamed to `startNtpClient`

Reply:
ok I see now; I would rename it as `forkNtpClient`.

Reply:
OK done